documentation/sdk/docs/getting-started/nodejs.mdx

```diff
@@ -0,0 +1,167 @@
+---
+image: /img/banner/client_lib_overview.png
+description: "Learn how to get started with the IOTA SDK in Node.js and build applications on the Shimmer network."
+keywords:
+- node.js
+- IOTA SDK
+- installation
+- client
+- wallet
+- usage
+- how-to guides
+- examples
+- development
+---
+
+import WarningPasswordStorage from '../_admonitions/_warning_password_storage.md'
+
+# Getting Started with Node.js
+
+<WarningPasswordStorage/>
+
+## Requirements
+
+- One of the following Node.js versions: '16.x', '18.x';
+
+## Install the IOTA SDK
+
+### Install Using a Package Manager
+
+To start using the IOTA SDK in your Node.js project, you can install the IOTA SDK from your package manager of choice:
+
+### npm
+
+```sh
+npm i @iota/sdk
+```
+
+### Yarn
+
+```sh
+yarn add @iota/sdk
+```
+
+### Build the Binding from Source
+
+#### Requirements
+
+If you want to build the IOTA SDK from source, please ensure you have the installed the following:
+
+* A [supported version of Node and Rust](https://github.com/neon-bindings/neon#platform-support).
+* Python < 3.11
+* Yarn v1
+
+#### Windows
+
+On Windows, you will also need an LLVM. Our workflow uses
+`https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.1/LLVM-11.0.1-win64.exe`. You may also need to set
+an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
+
+#### Build
+
+You can build the IOTA SDK from source by doing the following:
+
+1. Download the repository to any directory you choose:
+
+ ```shell
+ git clone https://github.com/iotaledger/iota-sdk
+    ```
+
+2. Move to the Node.js binding directory:
+
+ ```shell
+cd iota-sdk/bindings/nodejs
+    ```
+
+3. Install the necessary dependencies with either of the following commands:
+
+ ```shell
+ npm install
+    ```
+ ```shell
+ yarn install
+    ```
```

Comment:
Do both work? Do we want to promote both?
If so we should use tabs or [npm2yarn](https://docusaurus.io/docs/markdown-features/code-blocks#npm2yarn-remark-plugin)